src/admin/pages/Withdraw.vue

```diff
@@ -337,19 +337,27 @@ export default {
 
                 } else if ( 'bank' === method ) {
                     if ( data.bank.hasOwnProperty('ac_name') ) {
-                        details = this.sprintf( this.__( 'Account Name: %s', 'dokan-lite' ),  data.bank.ac_name );
+                        details = this.sprintf( this.__( '<p>Account Name: %s</p>', 'dokan-lite' ),  data.bank.ac_name );
```

Comment:
html tags in translatable string will throw js error for some languages, kindly remove p tags from all translatable strings